openmdao/solvers/linear/tests/test_direct_solver.py

```diff
@@ -546,7 +546,7 @@ def test_error_on_singular_with_densejac_bug(self):
         model.connect('c1.y', 'c2.x')
         model.connect('c2.y', 'c3.x')
         model.connect('c3.y', 'c4.x')
-        model.connect('c4.y', 'c5.x', src_indices=([0]))
+        model.connect('c4.y', 'c5.x', src_indices=([0]), flat_src_indices=True)
```

Comment:
Have you checked through the notebook docs to see if we need to add the flat_src_indices argument there to get rid of warnings?

Reply:
Yeah I checked.  No changes were needed.